pandas/plotting/_misc.py

```diff
@@ -363,17 +388,17 @@ def bootstrap_plot(series, fig=None, size=50, samples=500, **kwds) -> Figure:
 
 
 def parallel_coordinates(
-    frame,
-    class_column,
-    cols=None,
-    ax=None,
-    color=None,
-    use_columns=False,
-    xticks=None,
+    frame: DataFrame,
+    class_column: str,
+    cols: list[str] | None = None,
+    ax: Axes | None = None,
+    color: list[str] | tuple[str, ...] | None = None,
+    use_columns: bool = False,
+    xticks: Sequence | None = None,
```

Comment:
based on the docs, this should be `list | tuple | None`

Reply:
changed